tests/cypress/support/commands.js

```diff
@@ -36,6 +42,14 @@ Cypress.Commands.add("navigateToHomePage", () => {
   if (cy.location("pathname") !== "/") cy.visit("/");
 });
 
+Cypress.Commands.add("testPageAccessibility", () => {
+  Object.keys(breakpoints).forEach((deviceSize) => {
+    const size = breakpoints[deviceSize];
+    cy.viewport(...size);
+    cy.runAccessibilityTests();
+  });
```

Comment:
Suggestion: Use Object.values to skip the object lookup, and `for...of` over `.forEach`

```suggestion
  for (let size of Object.values(breakpoints)) {
    cy.viewport(...size);
    cy.runAccessibilityTests();
  }
```